src/library/autodj/dlgautodj.cpp

```diff
@@ -74,8 +74,14 @@ DlgAutoDJ::DlgAutoDJ(QWidget* parent,
     connect(pushButtonAutoDJ, SIGNAL(toggled(bool)),
             this, SLOT(toggleAutoDJButton(bool)));
 
-    introOutroCheckbox->setChecked(m_pAutoDJProcessor->getUseIntroOutro());
-    connect(introOutroCheckbox, &QCheckBox::stateChanged,
+    introOutroCombobox->addItem(tr("Do not use intro/outro markers"),
+                                static_cast<int>(AutoDJProcessor::IntroOutroUsage::None));
+    introOutroCombobox->addItem(tr("Use intro/outro for transition time (whichever is shorter)"),
```

Comment:
Any suggestions for better wording?

Reply:
Maybe get rid of "for transition time", so just "Use intro/outro (whichever is shorter)"